include/xrpl/protocol/detail/sfields.macro

```diff
@@ -144,6 +144,7 @@ TYPED_SFIELD(sfOutstandingAmount,        UINT64,    25, SField::sMD_BaseTen|SFie
 TYPED_SFIELD(sfMPTAmount,                UINT64,    26, SField::sMD_BaseTen|SField::sMD_Default)
 TYPED_SFIELD(sfIssuerNode,               UINT64,    27)
 TYPED_SFIELD(sfSubjectNode,              UINT64,    28)
+TYPED_SFIELD(sfEscrowedAmount,           UINT64,    28, SField::sMD_BaseTen|SField::sMD_Default)
```

Comment:
@shawnxie999 & @oleks-rip  & @sappenin This field will be used for PayChan as well. Do we want to come up with a better name?

Reply:
Maybe something like `LockedAmount`? `LockedAmount` was part of the original MPT spec actually, to account for locked funds in escrow or pay chan. But since MPT didn't support these features at the time, this field was removed.